numerics/elliptic_integrals.cpp

```diff
@@ -1378,7 +1425,7 @@ void Elbdj(double const phi,
 //
 //     Output: elk
```

Comment:
I am not sure this comment is very helpful anymore.

Reply:
Maybe it produces an elk...